notebooks/end-to-end-structured/labs/5b_deploy_keras_vertex_babyweight.ipynb

```diff
@@ -0,0 +1,456 @@
+{
```

Comment:
<p>Add back "Lab Task" or "Exercise" to help locate the exercises.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/206/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Done. Thanks.</p>